src/css/blocks/stars-content.css

```diff
@@ -0,0 +1,44 @@
+.stars-content {
+  height: 96px;
+  width: 100px;
```

Comment:
Don't hardcode container height and width

Reply:
Thank you for comment, I deleted a height but if I don't fixe a width I will have the problem like this :
![image](https://github.com/mate-academy/layout_stars/assets/160237096/7e2647ad-ce55-41e0-8e16-607c812315b3)

I modified anothers problems that you told me, thank you for your attention.